docs/modules/operation/pages/deep-dive/events/event-debugging.adoc

```diff
@@ -82,56 +80,48 @@ You can also define your own UEIs.
 | A short log message for the event browser (secure field by default).
 
 | -n (--nodeid)
-| Database ID of associated node (or use foreignSource, foreignId parameters).
+| Database ID of associated node (or use `foreignSource`, `foreignId` parameters).
 
 | -p (-parameter)
-| Parameter in key=value form.
-For example, `-p 'url \http://www.opennms.com/'`
+| Parameter in `key=value` form (for example, `-p 'url \http://www.opennms.com/'`).
 
 | -s (--service)
 | Name of the associated service.
 
 | -x (--severity)
 | Severity of the event: +
-1=Indeterminate +
-2=Cleared +
-3=Normal +
-4=Warning +
-5=Minor +
-6=Major +
-7=Critical +
+1 = Indeterminate +
```

Comment:
This is the info. needed earlier! We could add this as a general topic, perhaps in the events chapter. I'll create a Jira for it (NMS-15274).